compose/builders/greenpress.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:14.17.0
```

Comment:
Use node alpine like at the admin.
You should probably use 14.17-alpine for all other images too (instead of 14.5)